internal/binutils/binutils.go

```diff
@@ -39,6 +40,8 @@ type Binutils struct {
 	rep *binrep
 }
 
+var objdumpLLVMVerRE = regexp.MustCompile(`LLVM\sversion (([0-9]*)\.([0-9]*)\.([0-9]*)).*\s`)
```

Comment:
An llvm-objdump compiled from llvm-project source: `  LLVM version 11.0.0git` (10.0.0 has been released, so the git HEAD version is now 11.0.0git)

Internal llvm-objdump may use other version scheme like `  llvm version google3-trunk 111111111111111111111111111111111commit-id` (I guess the lowercase `llvm` is not intentional. I can fix it.)

Reply:
I can make the regex to match case insensitive but google3-trunk is troublesome. The code will look for a major version and use that to pick the latest one. Other option is to skip the search and just look for 
llvm-objdump, objdump, gobjdump 
in that order

This will also skip looking for llvm-objdump-ver

Reply:
I will also simplify the regex to only match the major version and ignore the rest.

Reply:
It is now `LLVM version google3-trunk`. If the regex parses a version (`google3-trunk`), the code can just assume the version is new enough.

Reply:
@aalexand Ok to do this check or should we do it in the internal version only?

Reply:
We'll check for `trunk` and accept it.